### PR TITLE
Fixes #5684: Generic method to raise a reinventory

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/5684

### CODE REVIEW

File: tree/30_generic_methods/rudder_inventory_trigger.cf [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+#
+# @parameter id     Id of the inventory (should be unique for each call)
+#
+# @class_prefix rudder_inventory_trigger
+# @class_parameter id
+
+bundle agent rudder_inventory_trigger(id)
+{
+  vars:
+      "flag_file"         string => "/opt/rudder/etc/force_inventory";
```
Comment (Fdall) [inline tree/30_generic_methods/rudder_inventory_trigger.cf:35]:
```suggestion
      "flag_file"         string => "${rudder_base]/etc/force_inventory";
```

File: tree/30_generic_methods/rudder_inventory_trigger.cf [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+# @name Rudder inventory trigger
+# @description Trigger an inventory on the agent
+# @action
+# @documentation
+#
+# Trigger a Rudder inventory. This will not run the inventory
+# immediately but next time the agent runs.
+#
+# @parameter id     Id of the inventory (internal identifier, needs to be unique for each use of the method)
```
Comment (ncharles) [inline tree/30_generic_methods/rudder_inventory_trigger.cf:27]:
```suggestion
# @parameter id     Id of reporting for this method (internal identifier, needs to be unique for each use of the method)
```

**Comment by amousset** [comment]
@ncharles updated

**Comment by amousset** [comment]
ping @ncharles

**Review by Fdall — APPROVED** [review]

**Pull request merged**